src/docfx/template/TemplateEngine.cs

```diff
@@ -149,9 +149,7 @@ public static TemplateEngine Create(Docset docset, RestoreGitMap restoreGitMap)
             }
 
             var theme = LocalizationUtility.GetLocalizedTheme(docset.Config.Template, docset.Locale, docset.Config.Localization.DefaultLocale);
-            var themePath = restoreGitMap.GetGitRestorePath(theme, docset.DocsetPath);
```

Comment:
don't remove this, it's really helpful for debugging if the restore with dependency lock works well.

Reply:
It is now printed here: https://github.com/dotnet/docfx/pull/5061/files#diff-77d5963be746079df8655d36a6a64e91R49